k8s/charts/openebs/templates/deployment-local-provisioner.yaml

```diff
@@ -12,6 +12,9 @@ metadata:
     openebs.io/version: {{ .Values.release.version }}
 spec:
   replicas: {{ .Values.provisioner.replicas }}
+  strategy:
+    type: {{ .Values.provisioner.upgradeStrategy }}
```

Comment:
Can this be changed to local PV provisioner. Also update the above replicas to use the localpvprovisioner (this has been fixed in upstream)

Reply:
I will update it